index.html

```diff
@@ -605,8 +721,8 @@ <h2>Issuer</h2>
       <section>
         <h2>Proofs (aka Signatures)</h2>
         <p>
-A <a>credential</a> MAY be made verifiable by adding the following
-<a>property</a>:
+A <a>credential</a> or a <a>presentation </a> MAY be made verifiable by
+adding the following <a>property</a>:
```

Comment:
I recommend changing to:

"In order for a credential or presentation to be made verifiable, the following property MUST be added:"